src/applications/gi/sass/partials/_gi-profile-page.scss

```diff
@@ -436,4 +436,9 @@
   .secondary-sco-list {
     padding-left: 0;
   }
+
+  .eyb-value-header {
```

Comment:
can you put this class inside the `.your-estimated-benefits` class?

Reply:
Yeah sure, let me move that real quick